sdk/storage/Azure.ResourceManager.Storage/tests/Helpers/StorageTestBase.cs

```diff
@@ -66,6 +72,19 @@ public async Task<ResourceGroup> CreateResourceGroupAsync()
                 });
             return operation.Value;
         }
+        public async Task<string> CreateValidAccountNameAsync(string prefix)
+        {
+            string accountName = "";
+            for (int i = 0; i < 10; i++)
+            {
+                accountName = Recording.GenerateAssetName(prefix);
+                StorageAccountCheckNameAvailabilityParameters parameter = new StorageAccountCheckNameAvailabilityParameters(accountName);
+                CheckNameAvailabilityResult result =await DefaultSubscription.CheckStorageAccountNameAvailabilityAsync(parameter);
+                if (result.NameAvailable==true)
```

Comment:
spaces and curly brackets

Reply:
Thanks for you review Arthur! The result.NameAvailable is a bool? variable rather than a bool, which means it can be null. Using if (result.NameAvailable) would cause compile error here. As for other comments, I have modified the code according to them.